cmd/root.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"fmt"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"os"
+	"semangit/internal/plugins/helm"
+	"semangit/internal/plugins/versionanalyzer"
+	"semangit/internal/utils"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "serve",
+	Short: "Serve semangit",
+	Long:  `With semangit you can check your versions!`, // TODO: complete this description
+	Run: func(cmd *cobra.Command, args []string) {
+		if err := serveSemanGit(cmd, args); err != nil {
```

Comment:
*Semangit* is a single name.
```suggestion
		if err := serveSemangit(cmd, args); err != nil {
```